### PR TITLE
libbeat: fix 'make update' rule for community beats

### PR DESCRIPTION
**Pull request by cyrilleverrier**
I added the same options in libbeat/scripts/generate_fields_docs.py as it is  in libbeat/scripts/generate_template.py

### CODE REVIEW

File: libbeat/scripts/generate_fields_docs.py [hunk 1]
```diff
@@ -92,19 +93,25 @@ def fields_to_asciidoc(input, output, beat):
...
-    if len(sys.argv) != 3:
-        print "Usage: %s beatpath beatname" % sys.argv[0]
-        sys.exit(1)
 
-    beat_path = sys.argv[1]
-    beat_name = sys.argv[2]
+    parser = argparse.ArgumentParser(
+        description="Generates the documentation for a Beat.")
+    parser.add_argument("path", help="Path to the beat folder")
+    parser.add_argument("beatname", help="The beat fname")
```
Comment (ruflin) [inline libbeat/scripts/generate_fields_docs.py]: s/fname/name/

**Comment by ruflin** [comment]
LGTM. Just one typo inside. I'm thinking about extracting the common parts in these scripts in the future so we only have to maintain it in one place. The scripts get more and more similarities.

**Comment by cyrilleverrier** [comment]
@ruflin : fixed and branch is now rebased on top of master. Thanks for your review.

**Comment by ruflin** [comment]
jenkins, test it

**Pull request merged**